lib/pdf_fill/forms/va21p527ez.rb

```diff
@@ -727,14 +729,14 @@ def expand_financial_acct(recipient, financial_acct, financial_accts)
         financial_accts
       end
 
+      def unfilled_multiline_acct?(acct_type, accts)
+        %w[socialSecurity salary].include?(acct_type) && accts.size < 2
+      end
+
       def zero_financial_accts(financial_accts)
         financial_accts.each do |acct_type, accts|
-          if accts.size.zero? && acct_type != 'additionalSources'
-            accts << {
-              'recipient' => 'Myself',
-              'amount' => 0
-            }
-          end
+          accts << DEFAULT_FINANCIAL_ACCT unless accts.any?
```

Comment:
are you trying to check for the empty array here with `accts.any?`? how about using `accts.present?`, `.any?` isn't actually checking for an empty array

Reply:
Had a fun read about `.any?` vs `.present?` vs `.blank?` vs `.empty?` I'm cool with `.present?` I'll update it